src/Routing/Traits/RunwayRoutes.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace DoubleThreeDigital\Runway\Routing\Traits;
+
+use DoubleThreeDigital\Runway\AugmentedRecord;
+use DoubleThreeDigital\Runway\Models\RunwayUri;
+use DoubleThreeDigital\Runway\Routing\ResourceResponse;
+use DoubleThreeDigital\Runway\Runway;
+use Statamic\Routing\Routable;
+use Statamic\View\Antlers\Parser;
+
+trait RunwayRoutes
+{
+    use Routable {
+        uri as routableUri;
+    }
+
+    public function route()
+    {
+        return $this->runwayUri->uri;
+    }
+
+    public function routeData()
+    {
+        return [];
```

Comment:
This should probably return something more than an empty array. The question is: what should that be?